### PR TITLE
misc: mark dependencies of entity classification computed artifact

### PR DESCRIPTION
**Pull request by alexnj**

### CODE REVIEW

File: core/computed/entity-classification.js [hunk 1]
```diff
@@ -98,5 +98,6 @@ class EntityClassification {
   }
 }
 
-const EntityClassificationComputed = makeComputedArtifact(EntityClassification, null);
+const EntityClassificationComputed = makeComputedArtifact(EntityClassification,
```
[the remark this answers is not in the document]
Comment (alexnj) [reply core/computed/entity-classification.js:101]: Can you explain? May be I'm reading [this](https://github.com/GoogleChrome/lighthouse/blob/e31c345a6727aecdd06711bb0446d8e7826bd01f/core/computed/computed-artifact.js#L30) wrong, but it seems to pick only those dependencies picked via `keys` supplied during `makeComputedArtifact`. I've updated the one remaining call that had more than the required two parameters passed, but it didn't seem to create any difference in sample.json.

[the remark this answers is not in the document]
Comment (connorjclark) [reply core/computed/entity-classification.js:101]: Ah, I forgot we did that part recently. All good

File: core/test/results/sample_v2.json [hunk 2]
```diff
@@ -7698,12 +7698,6 @@
         "duration": 100,
         "entryType": "measure"
       },
-      {
-        "startTime": 0,
-        "name": "lh:computed:EntityClassification",
-        "duration": 100,
```
Comment (paulirish) [inline core/test/results/sample_v2.json:7704]: this kinda suggests that the computed audit isnt being run....

seems odd.

Comment (adamraine) [reply core/test/results/sample_v2.json:7704]: It was being run multiple times no? This PR removes the unnecessary run.

**Pull request merged**